### PR TITLE
[next] Update test fixture version

### PR DESCRIPTION
**Pull request by ijjk**
Noticed in https://github.com/vercel/vercel/pull/11503 that this test fixture is incorrectly locked to a specific version when we don't want it to be.

### CODE REVIEW

**Review by ztanner — APPROVED** [review]

File: .changeset/gentle-sloths-rescue.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/next': patch
```
Comment (trek) [inline .changeset/gentle-sloths-rescue.md:2]: Not blocking, but this is probably doesn't require even a patch version bump

**Review by trek — APPROVED** [review]

File: .changeset/gentle-sloths-rescue.md (hunk 1 above)
Comment (ijjk) [reply .changeset/gentle-sloths-rescue.md:2]: Did we remove the label that tells the changeset to ignore test only changes? We tried on a previous PR and it was blocking merge

Comment (trek) [reply .changeset/gentle-sloths-rescue.md:2]: That isn't a thing I've ever heard of! I've always seen them required in my time here but running changeset with `--empty` will generate an empty one.

Comment (trek) [reply .changeset/gentle-sloths-rescue.md:2]: Or just manually delete the content between the `---`s

Comment (TooTallNate) [reply .changeset/gentle-sloths-rescue.md:2]: You can push an empty changeset to make the bot happy, but anyways, the changeset is not a required check, so it's not a requirement.

**Review by hhjmmmmmmmm — APPROVED** [review]

**Review by hhjmmmmmmmm — APPROVED** [review]

**Pull request merged**